chroma-manager/chroma_core/services/job_scheduler/command_plan.py

```diff
@@ -346,6 +346,12 @@ def _emit_transition_deps(self, transition, transition_stack = {}):
         transition_stack[transition.stateful_object] = transition.new_state
         log.debug("Updating transition_stack[%s/%s] = %s" % (transition.stateful_object.__class__, transition.stateful_object.id, transition.new_state))
 
+        # do nothing for a NOOP transition
+        if transition.old_state == transition.new_state:
+            log.debug("NOOP transition %s -> %s" % (transition.old_state, transition.new_state))
+            return None
+
+        route = transition.stateful_object.get_route(transition.old_state, transition.new_state)
```

Comment:
why is this line added?

Reply:
Mistakenly I would say.  I will push a fix for that.  Given that it just duplicated the line below it, remving it shouldn't have any effect, but it should be removed all the same.